packages/spark-core/base/datePicker.js

```diff
@@ -1,22 +1,55 @@
+/* global document window */
 import TinyDatePicker from 'tiny-date-picker';
 import getElements from '../utilities/getElements';
+import getArrowClass from '../utilities/getArrowClass';
+
+const testWidthForReadOnly = (element) => {
+  element.removeAttribute('readonly');
+  if (window.innerWidth && window.innerWidth < 768) {
+    element.setAttribute('readonly', '');
+  }
+};
+
+const bindUIEvents = (element) => {
+  const input = element.querySelector('input');
+  const overrideMinDate = element.getAttribute('data-sprk-min-date');
+  const overrideMaxDate = element.getAttribute('data-sprk-max-date');
+
+  testWidthForReadOnly(input);
+
+  const dp = TinyDatePicker(input, {
+    mode: 'dp-below',
+    lang: {
+      days: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+    },
+    min: overrideMinDate || '01/1/2008',
+    max: overrideMaxDate || '01/1/2068',
+
+    format(date) {
+      // TODO: this likely doesnt work right in safari
```

Comment:
Leftover todo in here. I don't remember this?

Reply:
i just added it, i'll resolve it soonish